code/src/components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+export const Header = () => {
+  return (
+  <>
+    <div className='logo-container'>
+      <img src="./images/logo.png" alt="logo"/>
+      <h1>Institute of Space Stuff (IoSS)</h1>
+      </div>
+      <div className='evacuation-plan-container'>
+      <h3>EVACUATION-PLAN-LEAVING-EARTH-123</h3>
```

Comment:
I ran Lighthouse on your page and accessibility rating turned out quite high. One thing it showed though was regarding the use of h3 here on this line saying that heading elements are not in a sequentially descending order. Lighthouse also said that properly ordered headings that do not skip levels convey the semantic structure of the page, making it easier to navigate and understand when using assistive technologies. ⚙️

The way I understand Lighthouse’s message here is that you skipped an h2 heading and went straight to h3. One suggestion from my part would be to change the heading to an h2 here, that way you’re no longer skipping a heading. 🙂

It’s always good to run the page through different audit services such as Lighthouse and WAVE. Here’s also a checklist of things to test for in terms of accessibility: https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility (see the bottom of the page).